tests/installation/welcome.pm

```diff
@@ -128,4 +124,3 @@ sub post_fail_hook {
 }
 
 1;
-# vim: set sw=4 et:
```

Comment:
Don't remove this here in a commit that is about something else. Better change it in all files and propose a PR